plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/PostgreSqlQueryRunner.java

```diff
@@ -65,6 +66,7 @@ public static DistributedQueryRunner createPostgreSqlQueryRunner(
                     .build();
 
             queryRunner.installPlugin(new TpchPlugin());
+            queryRunner.installPlugin(new LongRunningPlugin());
```

Comment:
Postgres connector has nothing to do with event listeners. Let's not change it (not mix responsibilities of query runners)